src/main/java/teammates/ui/controller/AdminEmailComposeSendAction.java

```diff
@@ -19,6 +19,7 @@
 
 public class AdminEmailComposeSendAction extends Action {
 
+    private String status;
```

Comment:
`statusMessageToAdmin` would be a better name. Also update the other instances.

Reply:
Done